File: Day-16/Pattern28.py
n = int(input())

def PatternTwentyEight(n):
    i = 1
    while(i<=n):
        start = n-i+1
        num = i
        while(start):
            print(num, end='')
            start -=1
            num += 1
        
        star = (i -1)*2
        while(star):
            print('*', end='')
            star-=1
        
        revStart = n-i+1
        revNum=n
        while(revStart):
            print(revNum, end='')
            revNum-=1
            revStart-=1
        
        print(end='\n')
        i+=1
        

    
PatternTwentyEight(n)

File: Day-17/Pattern28.js
function PatternTwentyEight(n) {
  let i = 1;
  while (i <= n) {
    let str = "";
    let start = n - i + 1;
    let num = i;
    while (start) {
      str += num;
      start--;
      num++;
    }

    let spaces = (i - 1) * 2;
    while (spaces) {
      str += "*";
      spaces--;
    }

    let start2 = n - i + 1;
    let num2 = n;
    while (start2) {
      str += num2;
      num2--;
      start2--;
    }

    console.log(str);
    i++;
  }
}

PatternTwentyEight(5);
